sites/partners/src/applications/PaperApplicationForm/sections/FormDemographics.tsx

```diff
@@ -50,16 +50,20 @@ const FormDemographics = () => {
 
       <GridCell>
         <ViewItem label={t("application.add.race")}>
-          <Select
-            id="application.demographics.race"
+          <FieldGroup
             name="application.demographics.race"
-            placeholder={t("t.selectOne")}
-            label={t("application.add.race")}
-            labelClassName="sr-only"
+            fields={Object.keys(raceKeys).map((rootKey) => ({
+              id: rootKey,
+              label: t(`application.review.demographics.raceOptions.${rootKey}`),
+              value: rootKey,
+              subFields: raceKeys[rootKey].map((subKey) => ({
+                id: subKey,
+                label: t(`application.review.demographics.raceOptions.${subKey}`),
+                value: subKey,
+              })),
+            }))}
+            type="checkbox"
```

Comment:
Should this and the FieldGroup in `review/demographics` be moved to a shared helper?

Reply:
At the moment we're not able to import anything from ui-components in shared-helpers due to some sort of build issue. So I added this commented out in helpers with the PR number that should fix it to return to.